cirq/_doc.py

```diff
@@ -44,4 +46,20 @@ def document(value: Any, doc_string: Optional[str] = None):
     """
     docs = DocProperties(doc_string=doc_string)
     RECORDED_CONST_DOCS[id(value)] = docs
+
+    ## this is how the devsite API generator picks up
+    ## docstrings for type aliases
+    if sys.version_info >= (3, 7):
```

Comment:
Is this a true restriction? After some mild googling (and https://docs.python.org/3.6/reference/datamodel.html) it looks like it was fine in 3.6. If you're catching attribute errors anyways, why the need for a version check?

Reply:
The difference is `Union`, `Any`, `List` and others will fail under 3.7. 
🤦  you are right about the error catching though :D